### PR TITLE
Fix test suite forward compatibility with upcoming EventLoop releases

### PR DESCRIPTION
**Pull request by clue**
Refs https://github.com/reactphp/event-loop/pull/102 and https://github.com/reactphp/event-loop/pull/138

### CODE REVIEW

File: tests/Query/ExecutorTest.php [hunk 1]
```diff
@@ -290,6 +291,13 @@ private function createConnectionMock($emitData = true)
         return $conn;
     }
 
+    private function createTimerMock()
+    {
+        return $this->getMockBuilder(
+            'React\EventLoop' . (interface_exists('React\EventLoop\TimerInterface') ? '' : '\Timer') . '\TimerInterface'
```
Comment (kelunik) [inline tests/Query/ExecutorTest.php]: interface_exists('React\EventLoop\TimerInterface') ? 'React\EventLoop\TimerInterface' : 'React\EventLoop\Timer\TimerInterface' would be more readable IMO.

Comment (clue) [reply tests/Query/ExecutorTest.php]: Which is what I used before committing this, but it results in a rather long line :) Any suggestions?

Comment (clue) [reply tests/Query/ExecutorTest.php]: Anyway, updated to also match https://github.com/reactphp/promise-timer/pull/31 :+1:

Comment (kelunik) [reply tests/Query/ExecutorTest.php]: I wouldn't care about a longer line if it's more readable and I don't have to think. Just format it with multiple lines if you care about that:

```php
interface_exists('React\EventLoop\TimerInterface')
    ? 'React\EventLoop\TimerInterface'
    : 'React\EventLoop\Timer\TimerInterface'
```

Apart from that, just drop older PHP versions and use `::class` ;-)

**Pull request merged**